frontend/src/app/Settings/Users/UsersSideNav.tsx

```diff
@@ -23,9 +23,39 @@ const UsersSideNav: React.FC<Props> = ({
 }) => {
   return (
     <div className="display-block users-sidenav">
-      <h3>Users</h3>
+      <h3 className="users-header">Users</h3>
       <ul className="usa-sidenav">
         {users.map((user: LimitedUser) => {
+          let statusText;
+          switch (user.status) {
```

Comment:
I remember some discussion of sorting this list by user status, @vz3 did that end up in the final designs?